ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -914,11 +938,45 @@ bool OwInterface::softTorqueLimitReached (const string& joint_name) const
           JointsAtSoftTorqueLimit.end());
 }
 
-vector<double> OwInterface::getEndEffectorFT () const
+std::vector<double> OwInterface::getEndEffectorFT () const
 {
   return m_end_effector_ft;
 }
 
+std::vector<std::string> OwInterface::getActiveFaults (std::string subsystem_name) const
+{
+  if (m_fault_dependencies_on){
+    return m_fault_dependencies->getActiveFaults(subsystem_name);
+  }
+  else{
+    ROS_WARN("COULD NOT USE ActiveFaults LOOKUP, FAULT DEPENDENCIES ARE TURNED OFF");
```

Comment:
I got this error to come up once, and it tends to flood the console. I suggest using a throttle. The following would print the warning only every 10 seconds. 
```suggestion
    ROS_WARN_THROTTLE(10, "COULD NOT USE ActiveFaults LOOKUP, FAULT DEPENDENCIES ARE TURNED OFF");
```

Reply:
I had no idea ROS_WARN_THROTTLE existed, that's really useful!